src/common/Mutex.cc

```diff
@@ -61,10 +61,11 @@ Mutex::Mutex(const std::string &n, bool r, bool ld,
       _register();
   }
   else {
-    // If the mutex type is PTHREAD_MUTEX_NORMAL, deadlock detection
-    // shall not be provided. Attempting to relock the mutex causes
-    // deadlock. If a thread attempts to unlock a mutex that  it  has not
-    // locked or a mutex which is unlocked, undefined behavior results.
+    // If the mutex type is PTHREAD_MUTEX_DEFAULT, attempting to recursively
+    // lock the mutex results in undefined behavior. Attempting to unlock the
+    // mutex if it was not locked by the calling thread results in undefined
+    // behavior. Attempting to unlock the mutex if it is not locked results in
+    // undefined behavior.
```

Comment:
@liupan1111  What is the difference you noticed ? Please clarify.

If Sentence2 is generalizing the locked state, then Sentence1 can be removed.